wpeverest/everest-forms
# Fix - Navbar design while switching to another language

### All Submissions:






### Changes proposed in this Pull Request:



This PR fixes the broken design issue while switching another language from English.

### How to test the changes in this Pull Request:

1. Change the language from English to any (Recommended language: Spanish).
2. Open the form builder screen.
3. Check the if the navbar design breaks or not.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Navbar design while switching to another language
